columns/columns_test.go

```diff
@@ -21,16 +21,16 @@ type foos []foo
 func Test_Column_MapsSlice(t *testing.T) {
 	r := require.New(t)
 
-	c1 := columns.ForStruct(&foo{}, "foo")
-	c2 := columns.ForStruct(&foos{}, "foo")
+	c1 := columns.ForStruct(&foo{}, "foo", "id")
+	c2 := columns.ForStruct(&foos{}, "foo", "id")
 	r.Equal(c1.String(), c2.String())
 }
 
 func Test_Columns_Basics(t *testing.T) {
```

Comment:
I think this is missing two tests:

- PK is `id` and the ID field can not be written but can be read.
- PK is `notid` and the ID field ca be written and read but `notid` can not be written.
 have a test where the PK is not `"id"` and you are not able to update that particular PK

Reply:
I added cases but not everything is possible like you expect, see https://github.com/gobuffalo/pop/issues/556